src/Conference/ConferenceItem.jsx

```diff
@@ -41,11 +41,11 @@ const ConferenceItem = ({ conference }) => {
   return (
     <StyledConferenceItem>
       <StyledCard>
-        <StyledCardHeading>{name}</StyledCardHeading>
-        <p>{topic}</p>
+        <StyledCardHeading role="header">{name}</StyledCardHeading>
```

Comment:
Do you not mean role="heading"? https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/heading_role